desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -255,6 +260,8 @@ public void onRemove(final InputColumn<?> sourceColumn) {
 
     private static final Logger logger = LoggerFactory.getLogger(AnalysisJobBuilderWindow.class);
     private static final ImageManager imageManager = ImageManager.get();
+    
+    public static final List<Func<AnalysisJobBuilderWindowImpl, JComponent>> PLUGGABLE_LABELS_BANNER_COMPONENTS = new ArrayList<>(0);
```

Comment:
I know this is how we've done it in other places, but I'm really no fan of exposing what should be internal structures like this. Can't we use the ServiceLoader interface, or some other proper solution instead?

At least hide it behind an `addRightPanelComponent()` or something like that.

The name also seems a bit weird. It's not a banner?